drivers/hmis/app/graphql/types/hmis_schema/ce_match_rule.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeMatchRule < Types::BaseObject
+    field :id, ID, null: false
+    field :name, String, null: false
+    field :type, HmisSchema::Enums::CeMatchRuleType, null: false, method: :rule_type
+    field :owner_type, String, null: false
+
+    def owner_type # todo @martha - discuss
+      # config = object.applicability_config.symbolize_keys
```

Comment:
I got a bit stuck here and would like to discuss. Match rule owner is polymorphic, it can be an opportunity, project, or organization. Match rule applicability is also configured in the `applicability_config` object which indicates whether it applies to a given funder or project type.

I had a half baked idea that maybe this new schema field should be a flexible `owner_string` and can return something like:
- "Applies to this opportunity"
- "Inherited from [project | organization]"
- "Applies to this project's type"
- "Applies to this project's funder(s)"

etc. But this seems a little fishy somehow, we don't often return whole strings that the UI should display from the backend.